jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.vert

```diff
@@ -9,4 +9,4 @@ void main(){
    texCoord = inTexCoord;
    vec4 pos = vec4(inPosition, 1.0);
    gl_Position = vec4(sign(pos.xy-vec2(0.5)), 0.0, 1.0);
-}
\ No newline at end of file
+}

```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR